src/libraries/System.Private.CoreLib/src/System/Enum.cs

```diff
@@ -453,14 +514,13 @@ private static bool TryParse(Type enumType, string? value, bool ignoreCase, bool
                 throw new ArgumentException(SR.Arg_MustBeEnum, nameof(TEnum));
             }
 
-            ReadOnlySpan<char> valueSpan = value.AsSpan().TrimStart();
-            if (valueSpan.Length == 0)
+            value = value.TrimStart();
+
+            if (value.Length == 0)
```

Comment:
Nit (to match same code in other overload):
```suggestion
            value = value.TrimStart();
            if (value.Length == 0)
```